### ISSUE TITLE
IOS 12 2fa Backup not download

### ISSUE THREAD

**Issue by tapanshah1** [horrorho/InflatableDonkey#102]
I got all device snapshot(ios 12, 2fa) and i am getting all information but at last i m getting asset list empty in AssetsClient.java.

while searching record for download : 
API :: POST https://p33-ckdatabase.icloud.com/api/client/record/retrieve HTTP/1.1
see attachment getting response from api

<img width="1005" alt="Screenshot 2019-07-23 at 6 59 29 PM" src="https://user-images.githubusercontent.com/25950340/61716096-07cdf980-ad7c-11e9-83b7-81394bd2fbba.png">
<img width="1005" alt="Screenshot 2019-07-23 at 6 59 29 PM" src="https://user-images.githubusercontent.com/25950340/61716346-90e53080-ad7c-11e9-9569-98bea4110ae7.png">

In response i ma getting like " "
i am getting <CloudKit.RecordRetrieveResponse> responses example "- execute() - requestsCT:reordered [, , , , , , , , , , , , , , , , , , ]"
" CloudKit.RecordRetrieveResponse[, , , , , , , , , , , , , , , , , , ]"

**Comment by vipinbeni** [comment]
could you please share your code

**Comment by tapanshah1** [comment]
I used same latest master branch code.
Just i added password+6Digitcode and making request and i got token after using token i am making all snapshot download request.
For 6 digit code  : In iphone Go to Setting > Tap on First top row > Tap on Password & security >  click on "Get Verfication Code" 
enter that code with password Ex. password123456

**Comment by Pink280z** [comment]
Unfortunately, the download of iOS 11.3/12/13 backups on accounts with 2fa has been broken since Oct 2018. See this thread:
https://github.com/horrorho/InflatableDonkey/issues/87

and the last we heard from the author:
https://github.com/horrorho/InflatableDonkey/issues/87#issuecomment-429831966

**Comment by tapanshah1** [comment]
> Unfortunately, the download of iOS 11.3/12/13 backups on accounts with 2fa has been broken since Oct 2018. See this thread:
> #87
> 
> and the last we heard from the author:
> [#87 (comment)](https://github.com/horrorho/InflatableDonkey/issues/87#issuecomment-429831966)

Yes, I know that but when it going with manifest ids but that manifest ids information not there or not found. see above attachment.
May be path changed or cloudkit header change i am trying all.

**Comment by ymohammad** [comment]
Hi @tapanshah1 , I tried downloading backup for non 2fa account as well. It is not working for iOS 12 backup. Did anyone try for non 2fA account backup download?
